src/frontend/src/app/services/alert.service.ts

```diff
@@ -23,28 +23,29 @@ export class AlertService {
         });
     }
 
+    // TODO: Fix delete button
     success(message: string, keepAfterNavigationChange = false) {
         this.keepAfterNavigationChange = keepAfterNavigationChange;
-        this.subject.next({ type: 'success', text: message });
+        this.subject.next({ type: 'success', text: message});
         setTimeout(function () {
             this.closeMessage();
-        }.bind(this), 3000);
+        }.bind(this), 10000);
```

Comment:
for success 5000 is good, for warning 10,000 but for error 15,000 atleast
preferred is close button addition but that can wait as other stuff is more imp!